nixpkgs_review/github.py

```diff
@@ -49,13 +49,13 @@ def approve_pr(self, pr: int) -> Any:
         "Approve a PR"
         print(f"Approving {pr_url(pr)}")
         return self.post(
-            f"/repos/NixOS/nixpkgs/pulls/{pr}/reviews", data=dict(event="APPROVE"),
+            f"/repos/NixOS/nixpkgs/pulls/{pr}/reviews", data=dict(event="APPROVE")
         )
 
-    def merge_pr(self, pr: int) -> Any:
+    def merge_pr(self, pr: int, data: Optional[Dict[str, str]] = None) -> Any:
```

Comment:
Expose this as an optional argument instead for more type safety:

```suggestion
    def merge_pr(self, pr: int, method_method: str = "merge") -> Any:
```